.test-infra/jenkins/LoadTestsBuilder.groovy

```diff
@@ -56,6 +49,14 @@ class LoadTestsBuilder {
     }
   }
 
+  static String setBQDatasetNameByTriggerContext(String baseName, TriggeringContext triggeringContext) {
```

Comment:
Please rename to `getBigQueryDataset`

Reply:
Please look at the discussion above

Reply:
Ok. I've seen it. I don't want to bikeshed about it much but just to explain: 
 - we're not setting anything. It's a static method that lets us get the name given the right context and base name. So set can be misguiding.
 - the method arguments already tell us what is being used to determine what the name should be (`baseName`, `triggeringContext`). Mentioning them (the context) in the name too is a noise IMO and should be avoided. 

This is why I suggested this. Let me know if you agree.

Reply:
Thanks for explanation. I think you have exhausted a topic and I believe we can agree that  `getBigQueryDataset` is better name for this method. And in addition it's much shorter.